src/handlers/canonicalize_issue_links.rs

```diff
@@ -15,11 +15,8 @@ use crate::{
     handlers::Context,
 };
 
-// Taken from https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue?quot#linking-a-pull-request-to-an-issue-using-a-keyword
-static LINKED_RE: LazyLock<Regex> = LazyLock::new(|| {
-    Regex::new("(?i)(?P<action>close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)(?P<spaces>:? +)(?P<issue>#[0-9]+)")
-        .unwrap()
-});
+static LINKED_RE: LazyLock<Regex> =
+    LazyLock::new(|| Regex::new(r"(?P<start> |^)(?P<issue>#[0-9]+)(?P<end>\b)").unwrap());
```

Comment:
`\b` is just an assertion, it doesn't match anything (https://regex101.com/r/W8p1ED/1), so we don't even need to capture and replace it, just have it at the end of the regex.

Reply:
Even better, thks.